UnitsNet/Scripts/UnitDefinitions/BrakeSpecificFuelConsumption.json

```diff
@@ -0,0 +1,43 @@
+﻿{
+  "Name": "BrakeSpecificFuelConsumption",
+  "BaseUnit": "KilogramPerJoule",
+  "XmlDoc": "Brake specific fuel consumption (BSFC) is a measure of the fuel efficiency of any prime mover that burns fuel and produces rotational, or shaft, power. It is typically used for comparing the efficiency of internal combustion engines with a shaft output.",
+  "Units": [
+    {
+      "SingularName": "GramPerKiloWattHour",
+      "PluralName": "GramsPerKiloWattHour",
+      "FromUnitToBaseFunc": "x/3.6e9",
+      "FromBaseToUnitFunc": "x*3.6e9",
+      "Localization": [
+        {
+          "Culture": "en-US",
+          "Abbreviations": [ "g/kWh" ]
+        }
+      ]
+    },
+    {
+      "SingularName": "KilogramPerJoule",
+      "PluralName": "KilogramsPerJoule",
+      "FromUnitToBaseFunc": "x",
+      "FromBaseToUnitFunc": "x",
+      "Localization": [
+        {
+          "Culture": "en-US",
+          "Abbreviations": [ "kg/J" ]
+        }
+      ]
+    },
+    {
+      "SingularName": "PoundPerHorsepowerHour",
```

Comment:
It's `MechanicalHorsePower` and `Punds` (as they are defined in the Power and Mass units). Changing the name of the unit.